qfieldsync/utils/file_utils.py

```diff
@@ -51,6 +51,9 @@ def path_to_dict(path: PathLike, dirs_only: bool = False) -> DirectoryTreeDict:
             if dirs_only and not subpath.is_dir():
                 continue
 
+            if "/.qfieldsync" in str(subpath):
```

Comment:
The initial `/` is on purpose?

Reply:
Doesn't really matter, this is slightly stricter. Will force change it without the slash so it does not raise questions anymore.